Packs/DSPM/Playbooks/playbook-DSPM_Remediation_for_Sensitive_asset_open_to_world.yml

```diff
@@ -0,0 +1,410 @@
+    separatecontext: false
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 930,
+          "y": 210
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 19e7a8bf-75a4-46fa-856e-d43a6125dd7a
+    type: regular
+    task:
+      id: 19e7a8bf-75a4-46fa-856e-d43a6125dd7a
+      version: -1
+      name: Slack block list overwritten with a success message
+      description: This automation script overwrites the value of a specified list and sends a Slack notification to inform the user that they failed to respond to an incident notification in a timely manner. The notification includes a message indicating the end of the incident playbook and an invitation to reopen the incident if necessary.
+      scriptName: DSPMOverwriteListAndNotify
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      list_name:
+        simple: ${block_list_name}
+      message:
+        simple: "For XSOAR incident ID : ${incident.id} successfully applied public access block to the ${incident_object.assetName} bucket. Also updated risk: ${incident.riskfindingid} status to 'INVESTIGATION'.\nThanks. "
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1560,
+          "y": 1040
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: f4bcb2f8-4688-4c65-8495-f809e6838bc1
+    type: condition
+    task:
+      id: f4bcb2f8-4688-4c65-8495-f809e6838bc1
+      version: -1
+      name: Check the Cloud Provider of asset
+      description: Check whether the values provided in arguments are equal. If either of the arguments are missing, no is returned.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      AWS:
+      - "1"
+    separatecontext: false
+    conditions:
+    - label: AWS
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: ${incident_object.cloudProvider}
+            iscontext: true
+          right:
+            value:
+              simple: AWS
+    - label: Azure
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: ${incident_object.cloudProvider}
+            iscontext: true
+          right:
+            value:
+              simple: AZURE
+    - label: GCP
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: ${incident_object.cloudProvider}
+            iscontext: true
+          right:
+            value:
+              simple: GCP
```

Comment:
This playbook doesn't contain any branches to handle the `Azure` or `GCP` conditions from this task.  Are those still in process to be developed?  If not, this conditional task is not needed and should be removed.

Reply:
We will add support for these branch.